<launch>
    <include file="$(find learning_tf)/launch/start_demo1.launch"/>

    <node name="turtle_tf_listener" pkg="learning_tf" type="turtle_tf_listener.py"
          respawn="false" output="screen">
      <param name="leader_tf" type="string" value="/turtle1"/>
    </node>
  </launch>
